### ISSUE TITLE
Rename general_question* to course_question*

### ISSUE THREAD

**Issue by karyon** [fsr-itse/EvaP#1161]
tl;dr: basically, search for `general_question[naire]` and change all occurrences (there shouldn't be too many) to `course_question[naire]` as approriate. Some edge cases may occur ;)

Each course has a `general_contribution`, which contains questionnaires that are not associated to any contributor.

The questionnaires are currently either used for contributions that have an actual contributor, or for the general_contribution. These questionnaires are called general_questionnaires or course_questionnaires in different places. This should be unified, and we agreed to use course_questionnaire everywhere.

Also I saw some place where the general/course questionnaires were called "general_questions" which is actually wrong because it's about questionnaires, not questions. this should also be renamed to course_questionnaires.

(rationale: after this change we have contributor questionnaires and course questionnaires, which makes a lot of sense. course questionnaires are used for the general contribution, which makes less sense, but is acceptable. Calling the general contribution "course contribution" for full consistency is a bit too weird.)

**Comment by janno42** [comment]
In consideration of the changes planned for #1047 I would now suggest changing the name to "general question".
In the future, `Course` will be representing a certain course (e.g. "Maths"), for which we will run one or multiple `Evaluation`s. A common use case will be evaluating exams and in that case there will be no "course questions" whatsoever, because the exam belongs to the course, but we're explicitly not asking any questions about the course itself in this evaluation.
Having "general questions"/"allgemeine Fragen" and "contributor questions"/"personenbezogene Fragen" would be a more generic terminology, better fitting the different constellations of evaluations I expect we'll be using in the future.
